app/containers/Avatar.js

```diff
@@ -23,7 +24,17 @@ const Avatar = React.memo(({
 		type, text, size, userId, token, avatar, baseUrl
 	});
 
-	let image = (
+	let image = emoji ? (
+		<Emoji
+			theme={theme}
+			baseUrl={baseUrl}
+			getCustomEmoji={getCustomEmoji}
+			isMessageContainsOnlyEmoji
+			customEmojis
+			literal={emoji}
+			emojiName={emoji.replace(/:/g, '')}
+		/>
+	) : (
```

Comment:
Hey @Prateek93a, as it's related to `Message`, I think you can do this logic inside `MessageAvatar` instead of here.

Reply:
@djorkaeffalexandre Thanks for the review, Sir. Also, I am not sure if I fully understood this. Should I move the logic to compute `image` inside `MessageAvatar`? The `Avatar` component is used in other views as well, so should I remove `image` logic?